.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/aurelia-tools/.eslintrc.json",
+  "extends": "./../../aurelia-tools/.eslintrc.json",
```

Comment:
It was looking for the file in wrong dir (gulp-lint), let me check this again...